caiman/source_extraction/volpy/atm.py

```diff
@@ -280,7 +280,7 @@ def denoise_spikes(trace, sampleRate, windowLength, superfactor, threshs=(.4, .6
 
                 tlimit = len(trace)
 
-                thre = threshold_sets[th]
```

Comment:
Let's revert this set of changes though; not its job to weigh in on something that's not a misspelling

Reply:
FWIW, `thre` is a frequent typo of `three, there, their, the` hence was detected. I will make it skip all "thre" but then you would miss the typos.

Reply:
dropping 6642c7f3ad6347c25c175d60670e8787d64957f3